specs/latest/2.0/index.html

```diff
@@ -2123,6 +2123,9 @@ <h2 class="no-toc">Table of contents</h2>
         </table>
         <p>If <em>pname</em> is not in the table above, generates an <code>INVALID_ENUM</code> error.</p>
         <p>If <em>sampler</em> is not a valid sampler object, generates an <code>INVALID_OPERATION</code> error.</p>
+        <p>Modifying a parameter of a sampler object affects all texture units to which that sampler object is bound.
+        Calling <code>TexParameter</code> has no effect on the sampler object bound to the active texture unit.
+        It will modify the parameters of the texture object bound to that unit.</p>
```

Comment:
I agree with Mo. It isn't necessary to duplicate the spec text from the ES 3.0 spec. Additionally, that proposed sentence is incorrect. TexParameter modifies the parameters of a texture object, regardless of whether a sampler is bound. If a sampler's currently bound (overriding the parameters of the texture object), the texture's parameters are modified, and the sampler is unbound, then the texture's newly modified parameters take effect.